flutter/test/default_integrations_test.dart

```diff
@@ -22,6 +22,7 @@ void main() {
 
   tearDown(() {
     _channel.setMockMethodCallHandler(null);
+    FlutterErrorIntegration.defaultOnError = null;
```

Comment:
this would not be necessary if: https://github.com/getsentry/sentry-dart/pull/345/files#r590360951